build/automation/etc/git-secrets/nhsd-rules-banned.regexp

```diff
@@ -2,7 +2,9 @@
 [0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}
 [a-z]{2}-[a-z-]*-[1,2,3]\.rds\.amazonaws\.com
 rds\.[a-z]{2}-[a-z-]*-[1,2,3]\.amazonaws\.com
+arn:aws:.*:.*:[0-9]{12}
 dynamodb\.[a-z]{2}-[a-z-]*-[1,2,3]\.amazonaws\.com
+[0-9]{12}\.dkr\.ecr\.[a-z-]*-[1,2,3]\.amazonaws\.com
```

Comment:
Could this instead be `[1-9]{12}\.dkr\.ecr\.[a-z-]*-[1,2,3]\.amazonaws\.com` to allow use to use 000000000 as a mock aws account number?

Reply:
nice idea but this would prevent valid account ids with a 0 somewhere in them, i think i can achieve the same by explicitly adding something to the allow list, also will make this change in te make devops repo, whats the process for then pulling changes through to here please?

Reply:
looking again due to this existing rule in nhsd-rules-allowed.regexp 
(000000000000|123456789012)
this is already working, have tested it too